client/components/theme/index.jsx

```diff
@@ -534,7 +552,11 @@ export class Theme extends Component {
 			return this.renderUpsell();
 		}
 
-		return <span>{ translate( 'Free' ) }</span>;
+		return (
+			<a href={ this.getThemePriceUrl() }>
```

Comment:
I think this solution works better if we can:

1) disable the click on the link so it's only navigable using the keyboard. We can do it with the following CSS:

```
a.disable-pointer-click:active {
  pointer-events: none;
}

``` 
2) disable the keyboard navigation when the theme is `Free`
Maybe with `e.preventDefault()` or just `tabindex="-1"`

Reply:
I think it might be flagged as another a11y issue if we don't provide a valid link 😞